packages/terra-time-input/src/TimeInput.module.scss

```diff
@@ -249,57 +249,60 @@
 
   .meridiem-button-group {
     margin-right: var(--terra-time-input-meridiem-button-group-margin-right, 0.5rem);
-  }
 
-  .meridiem-button {
-    background-color: var(--terra-time-input-meridiem-button-background-color, transparent);
-    border-color: var(--terra-time-input-meridiem-button-border-color, #c8cacb);
-    color: var(--terra-time-input-meridiem-button-color, #1c1f21);
-    font-size: var(--terra-time-input-meridiem-button-font-size, 1rem);
-    height: var(--terra-time-input-meridiem-button-height, 2.1428571429rem);
-    line-height: var(--terra-time-input-meridiem-button-line-height, 1);
-    outline: none;
-    padding-bottom: var(--terra-time-input-meridiem-button-padding-bottom, 0.286rem);
-    padding-top: var(--terra-time-input-meridiem-button-padding-top, 0.286rem);
+    .meridiem-button {
```

Comment:
What issue did the previous specificity of the meridiem button cause?

Reply:
the default button styles were being preferred over the meridiem button styles causing the screenshots to fail.